source/common/filter/config_discovery_impl.h

```diff
@@ -165,6 +156,40 @@ class DynamicFilterConfigProviderImpl : public DynamicFilterConfigProviderImplBa
   ThreadLocal::TypedSlot<ThreadLocalConfig> tls_;
 };
 
+// Implementation of a HTTP dynamic filter config provider.
+class HttpDynamicFilterConfigProviderImpl
+    : public DynamicFilterConfigProviderImpl<Server::Configuration::NamedHttpFilterConfigFactory,
+                                             Http::FilterFactoryCb> {
+public:
+  HttpDynamicFilterConfigProviderImpl(FilterConfigSubscriptionSharedPtr& subscription,
```

Comment:
Seems that this c'tor passes all parameters to the base class c'tor.
If so, you may be able to just use the base c'tor (see ["Inheriting constructors" section](https://en.cppreference.com/w/cpp/language/using_declaration).

Reply:
Good idea. Done!